test/backend/fallbacks_test.rb

```diff
@@ -191,6 +191,56 @@ def setup
   end
 end
 
+# See Issue #590
+class I18nBackendFallbacksSymbolReolveRestartsLookupAtOriginalLocale < I18n::TestCase
+  class Backend < I18n::Backend::Simple
+    include I18n::Backend::Fallbacks
+  end
+
+  def setup
+    super
+    I18n.backend = Backend.new
+    I18n.enforce_available_locales = false
+    I18n.fallbacks = [:root]
+    store_translations(:ak,
+                       'calendars' => {
+                         'gregorian' => {
+                           'months' => {
+                             'format' => {
+                               'abbreviated' => {
+                                 1 => 'S-Ɔ'
+                                 # Other months omitted for brevity
+                               }
+                             }
+                           }
+                         }
+                       })
+    store_translations(:root,
+                       'calendars' => {
+                         'gregorian' => {
+                           'months' => {
+                             'format' => {
+                               'abbreviated' => :"calendars.gregorian.months.format.wide",
+                               'wide' => {
+                                 1 => 'M01'
+                                 # Other months omitted for brevity
+                               }
+                             },
+                             'stand-alone' => {
+                               'abbreviated' => :"calendars.gregorian.months.format.abbreviated"
+                             }
+                           }
+                         }
+                       })
+  end
+
+  test 'falls back to original locale when symbol resolved at fallback locale' do
+    Thread.new do
```

Comment:
I am not particularly clear on why this needs a `Thread` wrapping. What's happening here to require that?

Reply:
Whoops. Copy-paste error from the above test (`I18nBackendFallbacksLocalizeTestWithMultipleThreads`).

Removed.